src/main/java/greencity/exception/CustomExceptionHandler.java

```diff
@@ -17,69 +17,104 @@
+     * @author Marian Milian
      */
     @ExceptionHandler(RuntimeException.class)
-    public final ResponseEntity handle(RuntimeException e, WebRequest request) {
+    public final ResponseEntity handle(RuntimeException ex, WebRequest request) {
         ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
     }
 
     /**
-     * Generated javadoc, must be replaced with real one.
+     * Method intercept exception {@link BadEmailOrPasswordException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @param request contain  detail about occur exception
+     * @return ResponseEntity witch  contain http status and body  with message of exception.
+     * @author Marian Milian
      */
     @ExceptionHandler(BadEmailOrPasswordException.class)
-    public final ResponseEntity handle(BadEmailOrPasswordException e, WebRequest request) {
+    public final ResponseEntity handle(BadEmailOrPasswordException ex, WebRequest request) {
         ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
         log.trace(exceptionResponse.getTrace());
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(exceptionResponse);
     }
 
     /**
-     * Generated javadoc, must be replaced with real one.
+     * Method intercept exception {@link BadRefreshTokenException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @param request contain  detail about occur exception
+     * @return ResponseEntity witch  contain http status and body  with message of exception.
+     * @author Marian Milian
      */
     @ExceptionHandler(BadRefreshTokenException.class)
-    public final ResponseEntity<?> handle(BadRefreshTokenException e, WebRequest request) {
+    public final ResponseEntity handle(BadRefreshTokenException ex, WebRequest request) {
         ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(exceptionResponse);
     }
 
     /**
-     * Generated javadoc, must be replaced with real one.
+     * Method intercept exception {@link BadEmailException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @return ResponseEntity witch  contain http status and body  with message of exception.
+     * @author Nazar Stasyuk
      */
     @ExceptionHandler(BadEmailException.class)
-    public final ResponseEntity handle(BadEmailException e, WebRequest request) {
+    public final ResponseEntity handle(BadEmailException ex) {
         ValidationExceptionDto validationExceptionDto =
-            new ValidationExceptionDto("email", e.getMessage());
-        log.trace(e.getMessage(), e);
+            new ValidationExceptionDto("email", ex.getMessage());
```

Comment:
"email" is a key for error? 
if so better to have separate constant or enum for keys.

Check out other places where ValidationExceptionDto is used.

> .body(Collections.singletonList(validationExceptionDto));